ext/LegendGeSimExt.jl

```diff
@@ -0,0 +1,567 @@
+module LegendGeSimExt
+using LegendGeSim
+using PropDicts
+using LegendGeSim
+using LegendGeSim: Environment
+using MJDSigGen
+using MJDSigGen: SigGenSetup
+using LegendGeSim: SiggenSimulator
+using ArgCheck
+using ArraysOfArrays
+using CurveFit
+using DelimitedFiles
+using Distributions
+using DSP 
+using ElasticArrays
+using HDF5
+using IntervalSets
+using JSON
+using LegendDataManagement
+using LegendDataTypes
+using LegendHDF5IO
+using LegendTextIO
+using LinearAlgebra
+using Parameters
+using Polynomials
+using PropDicts
+using RadiationDetectorDSP 
+using RadiationDetectorSignals
+using RadiationSpectra
+using Random
+using Random123
+using RecipesBase
+using Requires
+using SignalAnalysis
+using SolidStateDetectors
+using StaticArrays
+using Statistics
+using StatsBase
+using StructArrays
+using Tables
+using TypedTables
+using Unitful
+
+import LsqFit
+
+using SolidStateDetectors: ConstructiveSolidGeometry as CSG
```

Comment:
This list of packages could be skimmed down to only the required ones. But as long as no new package is introduced that is not part of LegendGeSim (except for `MJDSigGen`), this should be fine for now.

For example, `using SolidStateDetectors: ConstructiveSolidGeometry as CSG` is not needed here.